Gemfile.lock

```diff
@@ -202,6 +204,3 @@ DEPENDENCIES
   unicorn
   will_paginate_mongoid (~> 2.0)
   yajl-ruby
-
-BUNDLED WITH
```

Comment:
Is this ever useful information to have?  I don't know enough about the bundles.  If so, we don't necessarily need to delete this.